src/server.ts

```diff
@@ -66,6 +67,7 @@ app.get("/getPlacePhoto/:ref", getPlacePhoto);
 app.post("/promotePlace", promotePlace);
 app.post("/clickPromo", clickPromo);
 
+app.get("/getUser/:email", getUser);
```

Comment:
```suggestion
app.get("/getUser", getUser);
```

It would be better if we send the email through the request body instead of a param.  See all the changes I suggested on how to change your approach so we can do this.